.github/workflows/pr-comments.yaml

```diff
@@ -57,11 +57,11 @@ jobs:
           make dev/tools
           git checkout go.mod go.sum # This is because of some weirdness in `make dev/tools`
       - name: format
-        if: contains(github.event.comment.body, '/format')   # check the comment if it contains the keywords
+        if: contains(github.event.comment.body, '/format') || contains(github.event.comment.body, '@kumahq format')
         run: |
           make format
       - name: run golden_files
-        if: contains(github.event.comment.body, '/golden_files')  # check the comment if it contains the keywords
+        if: contains(github.event.comment.body, '/golden_files') || contains(github.event.comment.body, '@kumahq golden files')
```

Comment:
please keep command in 1 work `golden_files`

Reply:
Any particular reason why? dependabot just has them separate

Reply:
I'm afraid of conflicts but might be fine

Reply:
So what isn't supported here _anyway_ is something like `@kumahq format golden_files`. It'd require more advanced parsing of the comment body. You have to do: 

```
@kumahq format
@kumahq golden_files
```